code/cwe/RenderCsvData.hs

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env cabal
+{- cabal:
+build-depends: base, xml
+-}
```

Comment:
That's unusual

Reply:
Since it's just for one-shot codegen I figured it was not necessary to add this to the cabal file.